core/server/themes/loader.js

```diff
@@ -0,0 +1,39 @@
+var debug = require('debug')('ghost:themes:loader'),
+    config = require('../config'),
+    events = require('../events'),
+    read = require('./read'),
+    settingsApi = require('../api/settings'),
+    updateConfigAndCache,
+    loadThemes,
+    initThemes;
+
+updateConfigAndCache = function updateConfigAndCache(themes) {
+    debug('loading themes', themes);
+    config.set('paths:availableThemes', themes);
```

Comment:
Yes exactly. I want to get rid of adding themes to the config AND settings. However, this PR changes the behaviour (load only the active theme on init, all themes after the server starts). That PR will be a refactor only 😁